lib/bpk-appearance/src/BpkAppearance.js

```diff
@@ -17,38 +17,30 @@
  */
 
 /* @flow */
-
-import { initialMode, eventEmitter } from 'react-native-dark-mode';
-
-export type ColorSchemeName = 'light' | 'dark';
+import { Appearance } from 'react-native';
+import type { AppearancePreferences } from 'react-native/Libraries/Utilities/NativeAppearance';
 
 export type BpkAppearancePreferences = {
-  colorScheme?: ColorSchemeName,
+  colorScheme?: AppearancePreferences,
```

Comment:
`AppearancePreferences` type is the exact same as the previous self defined `ColorSchemeName` so utilising the platform type to ensure correctness